doc/api/tls.md

```diff
@@ -789,8 +790,6 @@ added: v0.11.3
 * `port` {number}
 * `host` {string}
 * `options` {Object}
-  * `host` {string} Host the client should connect to.
```

Comment:
Oh OK. So, on that page, there is one signature `tls.connect(options[, callback])` (which has port and host props) and then another one `tls.connect(port[, host][, options][, callback])` (which is where this change is). So I thought showing port and host was redundant.

If they're valid I will add them back. Do you think it's worth documenting what would happen if you passed in a host, then an options object with a different host defined?